### DISCUSSION TITLE
TI_PER_SEC constant ?

### DISCUSSION THREAD

**Discussion by Samuel-DEVULDER** [spotlessmind1975/ugbasic#273] [Ideas]
In this example, 

https://github.com/spotlessmind1975/ugbasic/blob/fcc1fe8076aa61254ed982278b8a6b93a025df79/examples/contrib_timers.bas#L16

We divide by 60. But that constant varies across platforms. For instance it is 50 for the MO5 (and only 10 by default on the TO7).

It would be nice if ugBasic introduces a platform-dependent constant containing the number of clock ticks (cf TI variable) per second. ==> `TI_PER_SEC`  constant

This is just like the [CLOCKS_PER_SEC](https://man7.org/linux/man-pages/man3/clock.3.html) in C.

**Comment by spotlessmind1975** [comment]
Hi @Samuel-DEVULDER, **you are right!**

Indeed, the BASIC of the 80s did not have these subtleties. And indeed some games were easier if they were programmed in NTSC and then played in PAL, as the game ran slower. Some time ago, I opened an issue on that #191 , so I put it on the "to do" list, with your suggestion as well.
